*** ./smali/com/android/server/SystemServer.smali	2017-04-26 08:02:04.470818999 +0000
--- /home/patchrom/device/temp/new_smali/services.jar.out/./smali/com/android/server/SystemServer.smali	2017-04-26 08:03:28.150780408 +0000
***************
*** 142,147 ****
--- 142,149 ----
      .param p0, "args"    # [Ljava/lang/String;
  
      .prologue
+     invoke-static {}, Lmiui/patchrom/ClassHook;->initServerHook()V
+ 
      new-instance v0, Lcom/android/server/SystemServer;
  
      invoke-direct {v0}, Lcom/android/server/SystemServer;-><init>()V
